src/UI/examples/Modal/RoundTrip/show_a_modal_which_cannot_be_closed_with_the_keyboard.php

```diff
@@ -7,7 +7,7 @@ function show_a_modal_which_cannot_be_closed_with_the_keyboard()
 
     $modal = $factory->modal()->roundtrip('My Modal 1', $factory->legacy('You cannot close this modal with the ESC key'));
     $modal = $modal->withCloseWithKeyboard(false);
-    $button1 = $factory->button()->standard('Open Modal 1', '#')
+    $button1 = $factory->button()->shy('Open Modal 1', '#')
```

Comment:
Is this change really intended? Because it affects an example from the Modals.

Reply:
Good catch, I will remove this.